Document/0x05a-Platform-Overview.md

```diff
@@ -510,14 +510,20 @@ Publishing an app is quite straightforward; the main operation is making the sig
 
 The Android application attack surface consists of all components of the application, including the supportive material necessary to release the app and to support its functioning. The Android application may be vulnerable to attack if it does not:
 
-- Validate all input by means of IPC communication or URL-schemes. See
+<<<<<<< HEAD
+- Validate all input by means of IPC communication or URL-schemes, see also:
   - [Testing for Sensitive functionality Exposure Through IPC](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-for-sensitive-functionality-exposure-through-ipc "Testing for Sensitive functionality Exposure Through IPC");
   - [Testing URL Schemes](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-custom-url-schemes "Testing URL Schemes").
+=======
+- Validate all input by means of IPC communication or URL-schemes. See
+  - [Testing for Sensitive functionality Exposure Through IPC](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-for-sensitive-functionality-exposure-through-ipc "Testing for Sensitive functionality Exposure Through IPC")
+  - [Testing URL Schemes](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-custom-url-schemes "Testing URL Schemes")
+>>>>>>> 7bb7e1ab70a6c8ae00fb0c3390c166d62dee89a8

```

Comment:
there are some merge conflicts or some rests of them

Reply:
Thanks, just fixed it.